app/javascript/packages/rails-i18n-webpack-plugin/extract-keys-webpack-plugin.js

```diff
@@ -17,7 +17,7 @@ const PLUGIN = 'ExtractKeysWebpackPlugin';
  *
  * @type {RegExp}
  */
-const TRANSLATE_CALL = /(?:^|[^\w'-])(?:I18n\.)?t\)?\(\s*['"](.+?)['"][,\s)]/g;
+const TRANSLATE_CALL = /(?:^|[^\w'-])t\)?\(\[?(['"][a-z\d\s_.,'"]+['"])]?[,\s)]/g;
```

Comment:
I hope we can continue to use regexes for this and don't have to get into parser/lexer/AST land lol

Reply:
Good luck to whoever has to try to interpret this regex in the future! (My future self included) 😅